dependencies/bionic/hammer_cli_foreman_puppet/control

```diff
@@ -0,0 +1,26 @@
+Source: ruby-hammer-cli-foreman-puppet
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: <>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               rake,
+               ruby-hammer-cli-foreman (<< 4.0.0),
+               ruby-hammer-cli-foreman (>= 3.0.0.pre.develop)
```

Comment:
I'm not sure if this is valid Debian versioning, but also a bit unsure what to recommend. Right now it's not installable anyway since there is no nightly Hammer package on Debian.

Reply:
```
15:35:51  pbuilder-satisfydepends-dummy : Depends: ruby-hammer-cli-foreman (>= 3.0.0.pre.develop) but 2.1.1-1 is to be installed
```

That's very weird because nightly should have a much newer package than 2.1.